api/client/container/update.go

```diff
@@ -0,0 +1,150 @@
+package container
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/docker/docker/api/client"
+	"github.com/docker/docker/cli"
+	runconfigopts "github.com/docker/docker/runconfig/opts"
+	containertypes "github.com/docker/engine-api/types/container"
+	"github.com/docker/go-units"
+	"github.com/spf13/cobra"
+)
+
+type updateOptions struct {
+	blkioWeight       *uint16
```

Comment:
Thanks @vdemeester but if we don't use pointers like
```
blkioWeight       *uint16
...
cpuShares         *int64
```
here, we may not be able to reference pointers in `resource` 
```
		BlkioWeight:       *opts.blkioWeight,
....
 		CPUShares:         *opts.cpuShares,
```
as well?

Maybe I missed something?